xcache/image-config.d/00-functions.sh

```diff
@@ -0,0 +1,25 @@
+supervisord_is_enabled () {
+    local service="$1"
+    [[ -e /etc/supervisord.d/${service}.conf ]]
+}
+
+
+supervisord_is_disabled () {
+    local service="$1"
+    [[ -e /etc/supervisord.d/${service}.conf.disabled ]]
+}
+
+
+supervisord_enable () {
+    local service="$1"
+    if supervisord_is_disabled ${service}; then
+        mv -f /etc/supervisord.d/${service}.conf.disabled /etc/supervisord.d/${service}.conf
+    fi
+}
+
+supervisord_disable () {
+    local service="$1"
+    if supervisord_is_enabled ${service}; then
+        mv -f /etc/supervisord.d/${service}.conf /etc/supervisord.d/${service}.conf.disabled
```

Comment:
Likewise (also up to you, take it or leave it)

```suggestion
        mv -f /etc/supervisord.d/${service}.conf{,.disabled}
```